### DISCUSSION TITLE
1

### DISCUSSION THREAD

**Discussion by ghost** [MortezaBashsiz/CFScanner#600] [Q&A]
1

**Comment by kingofkings2023** [comment]
سلام، بنده هم به نوبه خودم از مرتضی عزیز خواهش می کنم در صورت امکان اپدیت جدید رو منتشر کنن و مث همیشه توی اسکنرها در صدر یکه تازی کنن
به شخصه به تجربه هیچ اسکنری رو دست cfscanner ندیدم
